sdk/storage/storage-blob/src/BlobBatchClient.ts

```diff
@@ -285,7 +287,8 @@ export class BlobBatchClient {
    * console.log(batchResp.subResponsesSucceededCount);
    * ```
    *
-   * @example
+   * Example usage with a lease:
```

Comment:
Lets see if we can be consistent and use either `Example usage with` or `Example using`

Reply:
`Example using` means the same thing and is more concise, so I vote for that version.